index.html

```diff
@@ -180,5 +254,6 @@ <h5 class="form-heading">I'm always<br>interested in hearing about new projects,
         </ul>
         <img class="footer-img" src="assets/images/Shape.svg" alt="img"/>
     </footer>
+</div>
```

Comment:
- [ ] Impressive job @ismayito, I just have one suggestion that will make your design even better. Your contact section could be improved to be more responsive. Right now your design becomes a little different on a screen size `1000px` and below.  This is to make sure you have a webpage that is as close as the Figma design.

### Figma design 
![Screenshot 2022-12-07 203111](https://user-images.githubusercontent.com/69077061/206416814-9e5d6c4a-90c4-4a28-87c6-f5945ef9e270.png)

### Your design 
![Screenshot 2022-12-07 203211](https://user-images.githubusercontent.com/69077061/206416816-98c8b10c-1dcb-4953-93b4-144291bab053.png)